src/key_pair.rs

```diff
@@ -93,7 +97,7 @@ impl KeyPair {
 	/// Usually, calling this function is not neccessary and you can just call
 	/// [`from_der`](Self::from_der) instead. That function will try to figure
 	/// out a fitting [`SignatureAlgorithm`] for the given
-	/// key pair. However sometimes multiple signature algorithms fit for the
+	/// key pair. However, sometimes multiple signature algorithms fit for the
```

Comment:
nit: it would be nice to have these unrelated comment fixes in a separate commit.

Reply:
the same commit is fine as long as the number of unrelated fixes aren't too large.